pkg/scripts_test/config.go

```diff
@@ -17,9 +17,10 @@ type extensions struct {
 }
 
 type sumologicExtension struct {
-	InstallToken string            `yaml:"install_token"`
-	Tags         map[string]string `yaml:"collector_fields"`
-	APIBaseURL   string            `yaml:"api_base_url"`
+	InstallToken      string            `yaml:"install_token,omitempty"`
```

Comment:
should we remove the install_token entry?

Reply:
I need it to test backward compatibility